src/olympia/editors/views.py

```diff
@@ -424,8 +426,14 @@ def _queue(request, TableObj, tab, qs=None, unlisted=False,
         search_form = None
         is_searching = False
     admin_reviewer = is_admin_reviewer(request)
-    if not admin_reviewer and not is_searching and hasattr(qs, 'filter'):
-        qs = exclude_admin_only_addons(qs)
+    if hasattr(qs, 'filter'):
+        if waffle.switch_is_active('post-review'):
+            # Hide webextensions from the queues: auto-approve cron should pick
+            # them up.
```

Comment:
Maybe add to the comment that is happening to avoid a human reviewer picking it up